src/connector/src/sink/doris_starrocks_connector.rs

```diff
@@ -421,3 +561,107 @@ pub struct DorisInsertResultResponse {
     #[serde(rename = "ErrorURL")]
     err_url: Option<String>,
 }
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct StarrocksInsertResultResponse {
```

Comment:
Can we re-organize the codes to separate common codes, starrocks codes and doris codes into different files to make it clearer? 
For example,
- Put `Inserter`, `InserterBuilder`, `HeaderBuilder`, `InserterInner` into a separate file
- Put `DorisInsertResultResponse`, `DorisInserter` into doris.rs
- Put `StarrocksInsertResultResponse`，`StarrocksInserter` into starrocks.rs

Also, I think it is clearer to name `xxxInserter` to `xxxClient` and `xxxGet` to `xxxSchemaClient`. What do you think?